chaosaws/cloudwatch/probes.py

```diff
@@ -97,25 +106,25 @@ def get_metric_statistics(namespace: str, metric_name: str,
         )
 
 
-def get_metric_data(namespace: str, metric_name: str, dimension_name: str,
-                    dimension_value: str, statistic: str = None,
+def get_metric_data(namespace: str, metric_name: str,
+                    dimensions: List[Dict[str, Any]], statistic: str = None,
                     duration: int = 300, period: int = 60, offset: int = 0,
                     unit: str = None, configuration: Configuration = None,
                     secrets: Secrets = None) -> float:
     """Gets metric data for a given metric in a given time period. This method
-    allows for more data to be retrieved than get_metric_statistics
-
-    :params
-        namespace: The AWS metric namespace
-        metric_name: The name of the metric to pull data for
-        dimension_name: The name of the dimension to search for
-        dimension_value: The value to be used for searching the dimension
-        unit: The type of unit desired to be collected
-        statistic: The type of data to return.
-            One of: Average, Sum, Minimum, Maximum, SampleCount
-        period: The window in which to pull datapoints for
-        offset: The time (seconds) to offset the endtime (from now)
-        duration: The time (seconds) to set the start time (from now)
+    allows for more data to be retrieved than get_metric_statistics.
+
+        *namespace: The AWS metric namespace.
+        *metric_name: The name of the metric to pull data for.
+        *dimensions: Are expected as:
+            Name: The name of the dimension to search for.
+            Value: The value to be used for searching the dimension.
+        *unit: The type of unit desired to be collected.
+        *statistic: The type of data to return.
+            One of: Average, Sum, Minimum, Maximum, SampleCount.
+        *period: The window in which to pull datapoints for.
+        *offset: The time (seconds) to offset the endtime (from now).
+        *duration: The time (seconds) to set the start time (from now).
```

Comment:
Likewise, could these please be in the linked format above

Reply:
The style of the parameters was fixed according to your recommendations.